viewer-prototype/src/browser/trace-viewer/components/table-output-component.tsx

```diff
@@ -52,7 +52,7 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
     }
 
     renderMainArea(): React.ReactNode {
-        return <div id='events-table' className='ag-theme-balham-dark' style={{height: this.props.tableHeight, width: this.props.tableWidth}}>
+        return <div id='events-table' className='ag-theme-balham-dark' style={{height:this.props.style.height,width:this.props.widthWPBugWorkaround}}>
```

Comment:
You should keep the spaces here, after : and ,  This should be part of the linter and auto-fixed on save imho